kindly review and guide me if I missed something @tavisca-nmulani

tavisca-tgoyal opened the pull request.

tavisca-tgoyal committed:
Update Program.cs

tavisca-tgoyal committed:
modified the code little bit.

tavisca-nmulani commented on Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs:
```diff
@@ -28,20 +29,117 @@ static void Main(string[] args)
+            }
+
+
+            throw new NotImplementedException();
```
Please remove this throw exception code line

tavisca-nmulani commented on Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs:
```diff
@@ -28,20 +29,117 @@ static void Main(string[] args)
-            throw new NotImplementedException();
+        public static int[] SelectMeals(int[] protein, int[] carbs, int[] fat, string[] dietPlans)
+        {
+            
```
Please add validations on inputs as mentioned in constraints sections.

tavisca-tgoyal replied:
@tavisca-nmulani done!

tavisca-nmulani reviewed (changes requested):
Kindly make changes as suggested.
Will share a simplified code implementation which will help refactoring code

tavisca-tgoyal committed:
removed notImplementedException

tavisca-tgoyal committed:
added validations

@tavisca-nmulani added all the validations against all constraints defined in the problem definition.
